MarijaSaysHello/MarijaSaysHello.csproj

```diff
@@ -8,4 +8,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Folder Include="Task5\" />
```

Comment:
The folder is redundant here. You can create folder for the solution and move there new projects